packages/trader/src/App/Components/Elements/ContractAudit/contract-details.jsx

```diff
@@ -124,7 +138,7 @@ const ContractDetails = ({ contract_end_time, contract_info, duration, duration_
                             <ContractAuditItem
                                 id='dt_bt_label'
                                 icon={<Icon icon='IcContractPayout' size={24} />}
-                                label={localize('Payout per point')}
+                                label={vanilla_payout_text}
```

Comment:
Having payout per pip makes the condition `show_payout_per_point` at line 137 invalid, is it intended?

Reply:
It's not making it invalid, the logic for `vanilla_payout_text ` still shows `Payout per point` in all the cases, other than if the trade type belongs to vanilla FX. The condition is responsible for showing the label only for required trade types.